docs/getting-started-v2.md

```diff
@@ -0,0 +1,119 @@
+- oas:generate-java-client (voor JAVA client code)
+- oas:generate-netcore-client (voor .NET Core client code)
+- oas:generate-net-client (voor .NET Full Framework client code)
+- oas:generate-python-client (voor Python client code)
+
+Een lijst met andere ondersteunde generator opties kun je vinden in de [Generators List](https://openapi-generator.tech/docs/generators){:target="_blank" rel="noopener"} van OpenAPI Generator.
+
+Note. De prerequisite van OpenAPI Generator is JAVA. Je moet een JAVA runtime installeren voordat je OpenAPI Generator kunt gebruiken
+
+## Probeer en test de BrpProxy
+
+Door wettelijke restricties kan de BRP bevragen API geen informatie vragen leveren. Er wordt op dit moment aan gewerkt om deze restricties weg te halen. Totdat dit is gerealiseerd moet de BrpProxy worden gebruikt om de informatie vragen te kunnen krijgen.
+
+De BrpProxy is geïmplementeerd als een containerized applicatie, zodat het makkelijk kan worden getest op een lokale machine en worden gehost in een productie omgeving.
+
+Er is een referentie implementatie van een docker-compose configuratie te vinden in de repository die kan worden gebruikt om de BRP Proxy te draaien op een lokale machine.
+
+Voor test doeleinden is een mock van de RvIG variant van de BRP Persoon Bevragen API (BrpService) geïmplementeerd. Deze mock gebruikt de [testdataset persoonslijsten proefomgevingen GBA-V](https://www.rvig.nl/documenten/richtlijnen/2018/09/20/testdataset-persoonslijsten-proefomgevingen-gba-v) als input om de productie situatie zoveel mogelijk te kunnen simuleren.
+
+Ten behoeve van monitoring is Open Telemetry geïntegreerd in de BrpProxy. Met behulp van de docker-compose configuratie bestand wordt ook een Jaeger Container opgestart om de trace regels van de BrpProxy te visualiseren.
+
+Voor logging past de BrpProxy structured logging toe. Dit maakt het makkelijker voor tooling om de log regels te query-en en te filteren. Voor het visualiseren van de logging wordt Seq gebruikt.
+
+In de volgende paragrafen is beschreven hoe de BrpProxy  t.b.v. test doeleinden op een lokale machine kan worden geïnstalleerd en geconfigureerd.
+
+### Prerequisites
+
+- [Docker Desktop](https://www.docker.com/products/docker-desktop)
+
+### Bouwen van de BrpProxy Container Image
+
+- Clone de Haal Centraal BRP Persoon bevragen repository
+  ```sh
+  git clone https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen.git
+  ```
+- Switch naar de `develop` branch
+  ```sh
+  git checkout develop
+  ```
+- Bouw de BrpProxy Container Image. Dit kan enige tijd duren.
+  ```sh
+  docker-compose build
+  ```
+
+### Opstarten van de BrpProxy Container
+
+- Start de BrpProxy
+  ```sh
+  docker-compose up -d
+  ```
```

Comment:
In de discussie die je indertijd hebt opgestart moesten op dit punt nog allerlei cucumber opdrachten uitgevoerd worden. Dat is nu niet meer nodig?